ResultTests/ResultTests.swift

```diff
@@ -31,10 +31,21 @@ final class ResultTests: XCTestCase {
 	}
 
 	func testErrorsIncludeTheCallingFunction() {
-		let function = __FUNCTION__
-		XCTAssertEqual(Result<(), NSError>.error().function ?? "", function)
+//		let function = __FUNCTION__
+//		XCTAssertEqual(Result<(), NSError>.error().function ?? "", function)
```

Comment:
it segfault because of the usage of `.file` and `.function` so I guess it's :-1: